GliaWidgets/View/Chat/Entry/ChatMessageEntryView.swift

```diff
@@ -85,6 +88,13 @@ class ChatMessageEntryView: UIView {
         updateTextViewHeight()
     }
 
+    func updateLayout() {
+        environment.gcd.mainQueue.asyncAfter(.now() + 0.1) {
```

Comment:
Let's add comment explaining why this workaround is needed here.

Reply:
a76e9a4df03bc57953108d86ec9da7aa9bda9ce1